keyboards/eyeohdesigns/babyv/readme.md

```diff
@@ -0,0 +1,15 @@
+# babyv
+
+![babyv](https://i.imgur.com/iRUm9I5.jpg)
+
+A short description of the keyboard/project
+
+* Keyboard Maintainer: [Eye Oh! Designs](https://github.com/yourusername)
+* Hardware Supported: The PCBs, controllers supported
+* Hardware Availability: Links to where you can find this hardware
```

Comment:
You may want to change/update these to reflect the board in question.

Reply:
Fixed.